packages/connector-jsdom/src/connector.ts

```diff
@@ -293,9 +269,7 @@ export default class JSDOMConnector implements IConnector {
 
                     debug(`${this.finalHref} loaded, traversing`);
                     try {
-                        await this.server.emitAsync('traverse::start', event);
-                        await this.traverseAndNotify(window.document.children[0] as HTMLElement);
-                        await this.server.emitAsync('traverse::end', event);
+                        await this.parseHTML();
```

Comment:
Why can't we just do a `await this.server.emitAsync('can-traverse::html')`? I don't think we are doing anything similar with other parsers, right?

Reply:
You are right, I always forget that the `emitAsync` is going to wait until the listeners ends.